dashboard/test/controllers/rubrics_controller_test.rb

```diff
@@ -105,6 +105,8 @@ class RubricsControllerTest < ActionController::TestCase
     rubric = create :rubric
     learning_goal1 = create :learning_goal, rubric: rubric
     learning_goal2 = create :learning_goal, rubric: rubric
+    # TODO: save this to a variable and use it to create learning goal ai evaluations
+    create :rubric_ai_evaluation, user: student, requester: teacher, status: 1
```

Comment:
what breaks if you don't add this?

Reply:
nothing should break if that is not there. it is there simply because the model was added and part of that process for this PR is to start writing to it.